tests/functional/columns-from-file-old-output-manifest/expected/data/out/tables/sample2

```diff
@@ -0,0 +1,2 @@
+"col3","col1"
```

Comment:
Pokud jsou v manifestu vyjmenované sloupce, CSV nesmí mít header

Reply:
Takže se mění chování oproti tomu jak to bylo před tímto PR?

Reply:
Hm, fakt to tam bylo takhle. To je wierd. Zeptám se.

Reply:
https://keboolaglobal.slack.com/archives/C06LY0QJ3C5/p1722337731284179 tak to takhle nech ať neděláme BC break

Reply:
@novak-ji nic méně bych přidal do toho procesoru tu  možnost nastavit to hasHeader

Reply:
přidal jsem to nastavení + test